setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
```

Comment:
And maybe we should take care other `__init__.py`.
Can we use glob to match pattern of `__init__.py`?